coding-concepts.js

```diff
@@ -19,26 +19,30 @@ const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
 // a) Your answer:
+// 10 
 // b) Verify and explain:
-
+// Because .lenth returns the total number of elements by index and it's the last index plus 1.
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
 // a) Your answer:
+// o
 // b) Verify and explain:
-
+// Inputting in an argument of 4 specifies which index in the string to reference.
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+//console.log(languages[index])
 
 // a) Your answer:
+// "Ruby"
 // b) Verify and explain:
+// The variable index contains the number 1, so using that variable as the argument is the same as inputting a 1 into the argument to reference that particular index of the array.
```

Comment:
This is not a function call. So it's not an argument. It is actually abstracting the value at the index indicated. This code is calling on the index of the second value in the array because the variable "index" is assigned the number 1